src/game/GameStateMatch.cpp

```diff
@@ -96,11 +101,14 @@ void GameStateMatch::sync() {
 
 }
 
-void GameStateMatch::handle() {
+void GameStateMatch::handle(unsigned long countedFrames) {
```

Comment:
also why are you counting frames in a double (its already in the timer) and then casting it to a long?
that is really really bad. instead you should be taking in the stepTimer.getTicks() as whatever type it returns